### ISSUE TITLE
[Bug]: Button's with SVGs do not send name/values to the form data if the SVG is what is clicked

### ISSUE THREAD

**Issue by brookslybrand** [remix-run/remix#651]
### Which Remix packages are impacted?

- [ ] `remix` (Remix core)
- [ ] `create-remix`
- [ ] `@remix-run/architect`
- [ ] `@remix-run/cloudflare-workers`
- [ ] `@remix-run/dev`
- [ ] `@remix-run/express`
- [X] `@remix-run/netlify`
- [ ] `@remix-run/node`
- [X] `@remix-run/react`
- [ ] `@remix-run/serve`
- [ ] `@remix-run/server-runtime`
- [ ] `@remix-run/vercel`

### What version of Remix are you using?

1.0.5

### Steps to Reproduce

If you open [this example](https://stackblitz.com/edit/node-hwux9x?file=app/routes/index.tsx) and click the button, you'll see that the action function does not return the value of the button. If you click on the edges of the button (where the padding is), it will work. This can be seen in the gif below:

![svg-in-button-bug](https://user-images.githubusercontent.com/12396812/143487841-83e1bb3d-1f55-4b5b-8f91-a599c7d96524.gif)

### Expected Behavior

A button with an SVG in it should return the provided name/value of the button when clicked

### Actual Behavior

A button with an SVG does not return the provided name/value of the button when clicked

**Comment by nakleiderer** [comment]
Worth noting, it works fine with JavaScript disabled, so the bug is within the Form component (iinstead of the Browser).

**Comment by brookslybrand** [comment]
> Worth noting, it works fine with JavaScript disabled, so the bug is within the Form component (iinstead of the Browser).

Yes, thank you for pointing this out. I should have mentioned this disparate user experience
